tests/test_aea.py

```diff
@@ -101,6 +101,9 @@ def test_react():
         builder = AEABuilder()
         builder.set_name(agent_name)
         builder.add_private_key(FETCHAI, private_key_path)
+        builder.add_protocol(
```

Comment:
Hmm not a fan of making these tests heavier than necessary.

Reply:
from 0231f54:

Reply:
Hmm ok.